bin/run-command

```diff
@@ -119,87 +112,6 @@ task_arn=$(aws --no-cli-pager "${aws_args[@]}" --query "tasks[0].taskArn" --outp
 echo "::endgroup::"
 echo
 
-# Get the task id by extracting the substring after the last '/' since the task ARN is of
-# the form "arn:aws:ecs:<region>:<account id>:task/<cluster name>/<task id>"
-ecs_task_id=$(basename "${task_arn}")
-
-# The log stream has the format "prefix-name/container-name/ecs-task-id"
-# See https://docs.aws.amazon.com/AmazonECS/latest/userguide/using_awslogs.html
-log_stream="${log_stream_prefix}/${container_name}/${ecs_task_id}"
-
-# Wait for log stream to be created before fetching the logs.
-# The reason we don't use the `aws ecs wait tasks-running` command is because
-# that command can fail on short-lived tasks. In particular, the command polls
-# every 6 seconds with describe-tasks until tasks[].lastStatus is RUNNING. A
-# task that completes quickly can go from PENDING to STOPPED, causing the wait
-# command to error out.
-echo "Waiting for log stream to be created"
-echo "  task_arn=${task_arn}"
-echo "  task_id=${ecs_task_id}"
-echo "  log_stream=${log_stream}"
```

Comment:
Could we leave these in (feel free to change the message) - useful for grabbing the task ID and then going to cloudwatch yourself.

Reply:
👍🏽

Reply:
https://github.com/HHS/simpler-grants-gov/pull/4305/commits/5b132fc3ef8f9740e07ed0e88ff4ab1eb7ca01e7